lib/roust/ticket.rb

```diff
@@ -82,6 +82,35 @@ def ticket_update(id, attrs)
         end
     end
 
+    def ticket_comment(id, attrs)
+      attrs['Text'].gsub!(/\n/, "\n ") if attrs['Text'] # insert a space on continuation lines.
+      content = compose_content('ticket', id, attrs)
+
+      unless content.match(/Action: (comment|correspond)/i)
```

Comment:
The reason I haven't done this is that compose_content normalises the case and type of the keys.  Doing the Action check above would mean that the other fields could use symbols, and case-insensitive keys, but not Action.  That doesn't seem like a great user experience, and this seemed like a better solution.